web/src/components/profile/ProfileTab.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
 export const ProfileTab = (props: Props): ReactElement => {
   return (
     <button
-      className="cursor-pointer width-100 bg-base-lightest flex fjb fac padding-y-1 padding-right-2 padding-left-3 border-2px border-base-lighter btn-profile-hoverstate line-height-120"
+      className="cursor-pointer profile-tab-nav-button bg-base-lightest flex fjb fac padding-y-1 padding-right-2 padding-left-3 border-2px border-base-lighter btn-profile-hoverstate line-height-120"
```

Comment:
I tried to apply this concept higher in the tree but it was being overriden by things lower in the tree.